localisation/00_zuwc_united_localisation.csv

```diff
@@ -245,7 +245,7 @@ EVTDESC_WCBDP_50_alliance;Azeroth's nations are concerned about what is happenin
+EVTDESC_WCBDP_60_alliance;The Alliance Expedition crossed through the veil of the Dark Portal, and is ready to gain a foothold in this new wild world. Perhaps once beautiful, Draenor now looks more like a desolate wasteland, surrounded by boiling water from the contamination of the seas. Nevertheless, it is still possible to survive...Here, among the red sands, is one of the largest fortresses of the Horde, the Hellfire Citadel, where Ner'zhul tries to manipulate stolen artifacts of the orcs and open portals to other worlds. However, the expedition is unlikely to be have an easy conquest. Around the citadel are many orcish clans, ready to defend the fortress to the last. Some believe that it is much better to build a small bulwark near the Dark Portal, and to acquire a fleet to hit the orcs from the rear. Doing so will allow us to maintain an advantage in a pitched battle.;;Die Allianz-Expedition durchquerte den Schleier des dunklen Portals und ist bereit, in dieser neuen, wilden Welt Fu� zu fassen. Auch wenn Draenor einmal wundersch�n gewesen sein mag, �hnelt das Land jetzt eher einem verbrannten �dland, umgeben von brodelnden Meeren reinster Verderbnis. Trotzdem ist es immer noch m�glich, hier zu �berleben...\nHier, zwischen all dem roten Sand, steht eine der gr��ten Festungen der Horde - die H�llenfeuerzitadelle, in der Ner'zhul versucht, mit den gestohlenen Artefakten Portale zu weiteren, unschuldigen Welten zu �ffnen. Die Expedition wird kein leichtes Spiel haben - unz�hlige Ork-Clans lagern bei der Zitadelle und sind bereit, die Festung bis zuletzt zu verteidigen. Einige glauben, dass es besser w�re, eine kleine Festung in der N�he des dunklen Portals zu errichten, eine Flotte aufzustellen und die Orks zu umrunden und von hinten zu treffen, was es uns erm�glichte, einen Vorteil in der bevorstehenden Schlacht zu behaupten.;;;;;;;;;;;x
```

Comment:
English suggestions.
```suggestion
EVTDESC_WCBDP_60_alliance;The Alliance Expedition has crossed through the veil of the Dark Portal and is ready to gain a foothold on this new, untamed world. Perhaps it was beautiful once, but now Draenor has been rendered a desolate wasteland with boiling seas and shattered landmasses. Nevertheless, it is still somehow possible for life to survive... Here, among the red sands, is one of the largest fortresses of the Horde, Hellfire Citadel, where Ner'zhul attempts to utilize stolen artifacts of great power in order to open portals to other worlds. However, the expedition is unlikely to have an easy conquest. Around the citadel are several rage-fueled clans, ready to defend their fortress to the last orc. Many trust that it is wiser to build a bulwark near the Dark Portal; from there, a fleet can be used to flank the orcish forces. Doing this will allow us to maintain an advantage in an otherwise disadvantaged campaign.;;Die Allianz-Expedition durchquerte den Schleier des dunklen Portals und ist bereit, in dieser neuen, wilden Welt Fu� zu fassen. Auch wenn Draenor einmal wundersch�n gewesen sein mag, �hnelt das Land jetzt eher einem verbrannten �dland, umgeben von brodelnden Meeren reinster Verderbnis. Trotzdem ist es immer noch m�glich, hier zu �berleben...\nHier, zwischen all dem roten Sand, steht eine der gr��ten Festungen der Horde - die H�llenfeuerzitadelle, in der Ner'zhul versucht, mit den gestohlenen Artefakten Portale zu weiteren, unschuldigen Welten zu �ffnen. Die Expedition wird kein leichtes Spiel haben - unz�hlige Ork-Clans lagern bei der Zitadelle und sind bereit, die Festung bis zuletzt zu verteidigen. Einige glauben, dass es besser w�re, eine kleine Festung in der N�he des dunklen Portals zu errichten, eine Flotte aufzustellen und die Orks zu umrunden und von hinten zu treffen, was es uns erm�glichte, einen Vorteil in der bevorstehenden Schlacht zu behaupten.;;;;;;;;;;;x
```